crates/runtime/src/datafusion/udf/alias.rs

```diff
@@ -89,23 +73,10 @@ impl ScalarUDFImpl for ScalarUDFAlias {
         self.scalar_udf.is_nullable(args, schema)
     }
 
-    fn invoke_batch(
```

Comment:
Removed functions were deprecated and removed in DF47 in favor of single `invoke_with_args`